src/clients/SpokePoolClient.ts

```diff
@@ -21,7 +21,7 @@ import {
 } from "../utils/EventUtils";
 import winston from "winston";
 
-import { Contract, BigNumber, Event, EventFilter, ethers } from "ethers";
+import { BigNumber, Event, EventFilter, ethers } from "ethers";
```

Comment:
This is the 2nd import from ethers - candidate for consolidation? (also per your recent lint change in relayer-v2).

Reply:
Absolutely! This is a good call